### PR TITLE
Beta2

### PR DESCRIPTION
**Pull request by ctlong**
Mostly just style updates, but I have also added the possibility of a tie (where before if tied the game would go to player 2). Also updated the show and hide for the answer buttons so that the #question field can have more space when possible. Choosing an 'ok' answer now spawns an image until you choose the next question. I also edited my document format according to Harry's suggestions.

### CODE REVIEW

File: js/application.js [hunk 1]
```diff
@@ -190,10 +191,11 @@ $(document).ready(function() {
   function endGame() {
     if(clicked.length == 9) {
       var winner;
-      if(player1Score>player2Score) {winner = player1;}
-      else {winner = player2;}
+      if(player1Score>player2Score) {winner = player1+'! You have proven yourself the better wingman!!';}
```
Comment (h8rry) [inline js/application.js:194]: space between `>`, `+` and all the other operators

**Comment by h8rry** [comment]
Indentations haven't been fixed yet. Check out "convert indentations to spaces" in sublime

**Comment by h8rry** [comment]
It might "seem" like the indentations are correct in sublime. Check here https://github.com/WDI-HK-8/Wingman-Jeopardy/pull/2/files

**Comment by h8rry** [comment]
Thanks!

File: css/custom.css [hunk 2]
```diff
@@ -49,16 +49,16 @@ table {
   height: 250px;
 }
 
-.table-bordered>tbody>tr>td {
+.table-bordered > tbody > tr > td {
 	border: 1px solid black;
```
Comment (f3r) [inline css/custom.css:53]: fix indentation, 2 spaces :)

File: index.html [hunk 3]
```diff
@@ -52,14 +52,9 @@ <h3 id="player2">Player 2: 0</h3>
 				</table>
 			</div>
 			<footer class="col-xs-12">
-				<section class="col-xs-4"><h3 id="question">The number indicates the number of points you will receive upon a correct answer, once you choose a question you will have 10 seconds to answer.</h3></section>
-				<section class="col-xs-8">
-					<button class="btn btn-success answer"></button>
-					<button class="btn btn-success answer"></button>
-					<button class="btn btn-success answer"></button>
-				</section>
+				<section class="col-xs-12"><h3 id="question">The number indicates the number of points you will receive upon a correct answer, once you choose a question you will have 10 seconds to answer.</h3></section>
```
Comment (f3r) [inline index.html:55]:
``` html
<section class="col-xs-12">
  <h3 id="question">
    The number indicates the number of points you will receive upon a correct answer, once you choose a question you will have 10 seconds to answer.
  </h3>
</section>
```

File: js/application.js [hunk 4]
```diff
@@ -83,11 +81,13 @@ $(document).ready(function() {
   }
 
   function initAnswers(cur) {
+    $('footer img').remove();
+    $('footer>section').removeClass('col-xs-12').addClass('col-xs-4');
+    $('footer').append('<section class="col-xs-8"> <button class="btn btn-success answer"></button> <button class="btn btn-success answer"></button> <button class="btn btn-success answer"></button> </section>')
```
Comment (f3r) [inline js/application.js:86]: Would this be easier for someone else to read?

``` js
var html = "";
html += "<section class=\"col-xs-8\">":
html += "  <button class=\"btn btn-success answer\"></button>";
html += "  <button class=\"btn btn-success answer\"></button>";
html += "  <button class=\"btn btn-success answer\"></button>";
html += "</section>";

$('footer').append(html);
```

File: js/application.js [hunk 5]
```diff
@@ -83,11 +81,13 @@ $(document).ready(function() {
...
     for(var a=0;a<=2;a++) {
       var btn = $('.answer')[a];
       var str = 'answer'+(a+1)
       $(btn).text(cur[str]);
```
Comment (f3r) [inline js/application.js:90]: You could try something more like this:

``` js
$('.answer').forEach(function(btn, index) {
  $(btn).text( cur['answer'+index+1] );
}
```

Using `iterator functions` is generally a better idea than a `for loop` when you want to do something with all the elements of an array ;)

**Pull request closed without merging**